nbs/text.util.ipynb

```diff
@@ -386,25 +382,72 @@
     "    return result.replace(\"}{\", \" \")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "083e511c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# export\n",
+    "\n",
+    "\n",
+    "def text_to_sequence_for_editts(text, cleaner_names, symbol_set=GRAD_TTS_SYMBOLS):\n",
```

Comment:
can this be combined with the existing text_to_sequence easily?

Reply:
Not easily no. I initially tried to but it was a bit complicated so I just kept it as is for the sake of time